packages/block-directory/src/components/downloadable-blocks-list/index.js

```diff
@@ -59,78 +52,13 @@ function DownloadableBlocksList( {
 }
 
 export default compose(
-	withDispatch( ( dispatch, props ) => {
-		const { installBlock, downloadBlock } = dispatch(
-			'core/block-directory'
-		);
-		const { createErrorNotice, removeNotice } = dispatch( 'core/notices' );
-		const { removeBlocks } = dispatch( 'core/block-editor' );
-		const { onSelect } = props;
-
+	withDispatch( ( dispatch, { onSelect } ) => {
```

Comment:
While you're at it, you could refactor the component to just use the `useDispatch` hook instead of the `withDispatch` HOC.

Reply:
Good idea, thanks!